examples/chat_with_your_documents/README.md

```diff
@@ -0,0 +1,52 @@
+# Chat with your documents
+
+This folder contains a (very) minimal, self-contained example of how to make an application to chat with your documents, using Chroma and OpenAI's API.
+It uses the 2022 and 2023 U.S state of the union addresses as example documents.
+
+## How it works
+The basic flow is as follows:
+
+0. The documents in the `documents` folder are loaded, and embedded and stored line by line in a Chroma collection.
+
+1. When the user submits a question, it gets embedded using the same model as the documents, and the lines most relevant to the query are retrieved by Chroma.
+2. The user-submitted question is passed to OpenAI's API, along with the extra context retrieved by Chroma. The OpenAI API generates generates a response.
+3. The response is displayed to the user, along with the lines used as extra context.
+
+## Running the example
+
+You will need an OpenAI API key to run this demo. You can [get one here](https://platform.openai.com/account/api-keys).
+
+Install dependencies and run the example:
+
+```bash
+# Install dependencies
+pip install -r requirements.txt
+
+# Load the example documents into Chroma
+python load_documents.py
+
+# Run the chatbot
+python main.py
+```
+
+Example output:
+
+```
+Query: What was said about the pandemic?
```

Comment:
i get this ugly debug stuff about tokenizers 

<img width="1280" alt="Screenshot 2023-08-03 at 10 59 42 AM" src="https://github.com/chroma-core/chroma/assets/891664/71ffb635-61e6-46bd-85fd-
26a42ae655f7">

Reply:
yeah it's something to do with HF tokenizers, I don't know how to make it go away. It only happens the first time.